Nixie/Nixie.c

```diff
@@ -803,12 +809,16 @@ void display_task(void)
 				rgb_colour = YELLOW;
 			}
 			// NOTE: LEFT_DP1..LEFT_DP1 are contained in upper 8 bits!
-			if (p.sec & 0x01) nixie_bits  |=  RIGHT_DP4;
-			else              nixie_bits  |=  LEFT_DP5;
-			if (p.min & 0x01) nixie_bits  |=  RIGHT_DP2;
-			else              nixie_bits8 |=  LEFT_DP3;
-			if (dst_active)   nixie_bits  |=  RIGHT_DP6;
-			else              nixie_bits  &= ~RIGHT_DP6;
+			//if (p.sec & 0x01) nixie_bits  |=  RIGHT_DP4;
+			//else              nixie_bits  |=  LEFT_DP5;
+			nixie_bits  |=  LEFT_DP5;							// For IN-12B NIXIES the DP lits continue

```

Comment:
Prima als je die IN-12B continu wilt laten branden, maar je wilt natuurlijk op die andere Nixies wel die seconden laten verspringen. Dus moeten we die code er toch gewoon in houden?